x-pack/plugins/serverless_search/public/types.ts

```diff
@@ -32,6 +33,7 @@ export interface ServerlessSearchPluginStartDependencies {
   cloud: CloudStart;
   console: ConsolePluginStart;
   searchPlayground: SearchPlaygroundPluginStart;
+  searchInferenceEndpoints: SearchInferenceEndpointsPluginStart;
```

Comment:
you need to add `searchInferenceEndpoints` to `serverless_search/kibana.jsonc` file if you want this to be available here and it should be either optional or required. If it's optional (it should be) this should be marked as possibly undefined.

```suggestion
  searchInferenceEndpoints?: SearchInferenceEndpointsPluginStart;
```

Reply:
Updated